src/main/scala/firrtl/passes/ReplSeqMem.scala

```diff
@@ -88,7 +88,7 @@ Optional Arguments:
     error("No circuit name specified for ReplSeqMem!" + usage)
   )
   val target = CircuitName(passCircuit)
-  def duplicate(n: Named) = this copy (t = (t replace (s"-c:$passCircuit", s"-c:${n.name}")))
+  def duplicate(n: Named) = this copy (t = t replace(s"-c:$passCircuit", s"-c:${n.name}"))
```

Comment:
The deletion of the space after replace confuses me a bit. I think this might be illustrating another style-bug throughout Firrtl. I believe that you're only supposed to invoke methods using spaces if:

1. They are Arity-1:  Infix notation, eg. `names mkString ","`, and this should only be used for purely-functional methods or methods which take functions as parameters
2. They are higher-order functions, eg. `map` and `foreach`

replace is arity-2 and not higher-order so I think the right invocation may be `t.replace(...)`

This might also affect the style of copy since I do not believe it accepts high-order functions so it should generally be invoked with `.` syntax. Anyone have thoughts?

Source: http://docs.scala-lang.org/style/method-invocation.html